src/Chainweb/BlockHeader.hs

```diff
@@ -276,53 +279,106 @@ powTarget
         --
     -> HashTarget
         -- ^ POW target of new block
-powTarget p@(ParentHeader ph) bct
-    = case effectiveWindow ph of
-        Nothing -> maxTarget
-        Just w
-            | slowEpochGuard ver (_blockHeight ph) && slowEpoch p bct ->
-                adjust ver w (t .-. _blockEpochStart ph) (_blockTarget ph)
-            | isLastInEpoch ph ->
-                adjust ver w (t .-. _blockEpochStart ph) (_blockTarget ph)
-            | ver == Development && _blockHeight ph == 1 -> HashTarget (maxBound `div` 10000)
-                -- this is a special case for starting  new devnet. Using
-                -- maxtarget results in an two high block production and
-                -- consecutively orphans and network congestion. The consequence
-                -- are osciallations to take serval hundred blocks before the
-                -- system stabilizes. This code cools down initial block
-                -- production.
-            | otherwise -> _blockTarget ph
+powTarget p@(ParentHeader ph) bct = case effectiveWindow ph of
+    Nothing -> maxTarget
+    Just w
+        -- A special case for starting a new devnet. Using maxtarget results in
+        -- a too high block production and consecutively orphans and network
+        -- congestion. The consequence are osciallations to take serval hundred
+        -- blocks before the system stabilizes. This code cools down initial
+        -- block production.
+        | ver == Development && _blockHeight ph == (genesisHeight ver cid + 1) -> HashTarget (maxBound `div` 100000)
```

Comment:
```suggestion
        | ver == Development && _blockHeight ph == ((genesisHeight ver cid) + 1) -> HashTarget (maxBound `div` 100000)
```

Reply:
Also this seems like the only change yes? The rest is reformatting/reordering of cases?

Reply:
Why should the parenthesis be added? There is no other operator that could cause a precedence issue. Also, I think hlint would probably complain (although, I don't care that much about hlint...).

> Also this seems like the only change yes? The rest is reformatting/reordering of cases?
Yes

Reply:
It's just confusing to read, but I can live with it. In general I don't like arithmetic code that relies heavily on precedence for comprehension. In this case it wasn't clear if it was `(cid + 1)` as opposed to `(genesisHeight... + 1)`. Resolving